tests/Resources/LtiOutcome/valid_replace_result_response_body.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<imsx_POXEnvelopeResponse xmlns="http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0">
+    <imsx_POXHeader>
+        <imsx_POXResponseHeaderInfo>
+            <imsx_version>V1.0</imsx_version>
+            <imsx_messageIdentifier>e36f227c-2946-11e8-b467-0ed5f89f718b</imsx_messageIdentifier>
+            <imsx_statusInfo>
+                <imsx_codeMajor>success</imsx_codeMajor>
+                <imsx_severity>status</imsx_severity>
+                <imsx_description>Assignment with Id 1 was updated</imsx_description>
+                <imsx_messageRefIdentifier>1</imsx_messageRefIdentifier>
+                <imsx_operationRefIdentifier>replaceResult</imsx_operationRefIdentifier>
+            </imsx_statusInfo>
+        </imsx_POXResponseHeaderInfo>
+    </imsx_POXHeader>
+    <imsx_POXBody>
+        <replaceResultResponse />
+    </imsx_POXBody>
+</imsx_POXEnvelopeResponse>
```

Comment:
Please add empty line at the end of file

Reply:
Done, but then had to add trim when getting its content